src/util.h

```diff
@@ -26,6 +26,19 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 #pragma once
 
+#include <time.h>
```

Comment:
Don't think this include was needed just for the `#ifdef`, but it's fine, don't care much about it.

Reply:
Don't want the header to depend on headers it doesn't include or on the order it is included in a .c file.